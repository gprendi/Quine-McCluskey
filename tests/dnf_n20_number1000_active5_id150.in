d 20
00010000010001000000
00000000000000010000
00011000000000010000
00001000010000010000
00010000010001010000
00000000000000000000
00011000010001000000
00011000010001010000
00001000000000010000
00010000000000000000
00001000010001000000
00000000010000010000
00011000010000000000
00000000010000000000
00010000000001010000
00001000000000000000
00011000000000000000
00001000010000000000
00010000000000010000
00000000010001000000
00011000000001010000
00000000010001010000
00001000000001000000
00011000000001000000
00001000010001010000
00001000000001010000
00010000010000000000
00011000010000010000
00000000000001000000
00010000000001000000
00010000010000010000
00000000000001010000
