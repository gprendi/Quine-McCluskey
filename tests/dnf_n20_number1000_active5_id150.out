p dnf 20 0
-1 -2 -3 -6 -7 -8 -9 -11 -12 -13 -15 -17 -18 -19 -20 0
